src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -1944,5 +1944,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public Shadows Function SpansPreprocessorDirective(tokens As IEnumerable(Of SyntaxToken)) As Boolean
             Return MyBase.SpansPreprocessorDirective(tokens)
         End Function
+
+        Public Function IsLambdaBody(node As SyntaxNode) As Boolean Implements ISyntaxFactsService.IsLambdaBody
+            If node.Parent IsNot Nothing And TypeOf node.Parent Is LambdaExpressionSyntax Then
+                Dim lambdaNode = CType(node.Parent, LambdaExpressionSyntax)
+                Dim lambdaBodies = lambdaNode.GetBodies()
```

Comment:
how does a lambda node have 'bodies'?

Reply:
In VB you can have multiple line lambda and the unfortunate property of the Syntax tree there is that they're all under the `MultiLineLambdaExpressionSyntax` without being in any block.

Reply:
![image](https://user-images.githubusercontent.com/1764721/60376371-61cce080-99c3-11e9-8966-2f95c792d972.png)

Reply:
sorry, still not getting it.  the "pural" part of "bodies" is the part i'm not getting.

Reply:
For the lambda shown above both the `AssignementStatement` and `ReturnStatement` are part of what gets returned as Bodies (at least if I understood it correctly).  And it's not limited to two / one & return. If there was another line of `x += 20` between `x+=10` and the return then the Lambda would have 3 "bodies".